src/components/ApplicationPrompt.astro

```diff
@@ -0,0 +1,72 @@
+---
+import { addDays, DateFormatters } from '~utils';
+export interface Props {
+	cohortKickoffDate: Date;
+}
+
+const { cohortKickoffDate } = Astro.props as Props;
+
+// The application window is open for 1 week,
+// starting 2 weeks before the cohort kickoff
+// and ending 1 week before the cohort kickoff
+// (we're using 15 days to accout for Christmas this quarter)
+const [windowStart, windowEnd] = [
+	addDays(cohortKickoffDate, -15),
+	addDays(cohortKickoffDate, -7),
+];
+
+const applicationsOpen = windowStart >= new Date();
+
+const formattedKickoff = DateFormatters.fullDateLongMonth(cohortKickoffDate);
+const [formattedWindowStart, formattedWindowEnd] = [
+	DateFormatters.dayMonth(windowStart),
+	DateFormatters.dayMonth(windowEnd),
+];
+---
+
+<p class="u-text-large">
+	Our next cohort will kick off on {formattedKickoff} We will run teams this time
+	in North America and Europe/Africa time zones. The application window will be open
+	from {formattedWindowStart} to
+	{formattedWindowEnd}. We will notify you of your status the following week.
+</p>
+<>
+	{
+		!applicationsOpen ? (
+			<p class="u-text-large">
+				<a href="https://airtable.com/shrv9cqaTVHNhCvT1" class="c-button">
+					Apply now!
+				</a>
+			</p>
+		) : (
+			<>
+				<p class="u-text-bold u-text-large">
+					Return to this page between {formattedWindowStart} and{' '}
+					{formattedWindowEnd} to submit your application
+				</p>
+				<p class="u-text-small">
+					💡 Want a reminder of our next application window?
+					<a
+						href="/ics/Q1-2023-Application-Window.ics"
```

Comment:
Beyond the scope of this PR, but I'd love it if we can dynamically generate this `.ics` file in the future!